EssentialFeed/EssentialFeed/Image Comments/Image Comments Presentation/ImageCommentsViewModel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  ImageCommentsViewModel.swift
+//  EssentialFeed
+//
+//  Created by Lukas Bahrle Santana on 25/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public struct ImageCommentsViewModel {
+	public let imageComments: [PresentableImageComment]
+	
+	public init(imageComments: [PresentableImageComment]){
+		self.imageComments = imageComments
+	}
+}
+
+
+public struct PresentableImageComment: Hashable {
```

Comment:
Too many spaces - should be just one.

```suggestion
}

public struct PresentableImageComment: Hashable {
```